### PR TITLE
refactor(peers): remove references to `/connections`

### PR DESCRIPTION
**Pull request by ramfox**
* remove all references to `/connections`
* remove any hardwired `profile["online"] = true` statements

This pr also brings a bunch of node_packages up to date :)

### CODE REVIEW

File: app/dist/style.css [hunk 1]
```diff
@@ -200,7 +200,7 @@ html {
 html {
   font-size: 16px;
   -ms-overflow-style: scrollbar;
-  -webkit-tap-highlight-color: transparent; }
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0); }
```
Comment (b5) [inline app/dist/style.css:203]: reason for this?

File: app/dist/style.css [hunk 2]
```diff
@@ -381,7 +381,8 @@ html:hover [class^="ss-"] {
   -ms-font-feature-settings: "liga" 1;
   -o-font-feature-settings: "liga";
   font-feature-settings: "liga";
-  -webkit-font-smoothing: antialiased; }
+  -webkit-font-smoothing: antialiased;
```
Comment (b5) [inline app/dist/style.css:384]: OooOoooooh fancy

File: package.json [hunk 3]
```diff
@@ -135,7 +135,7 @@
   },
```
Comment (b5) [inline package.json:135]: in the future we should probs try to keep package updates as separate PR's so `git blame` points to an easy-to-isolate set of commits, but no biggie

**Review by b5 — APPROVED** [review]
🍩 🐶 🚯

File: app/dist/style.css (hunk 1 above)
Comment (ramfox) [reply app/dist/style.css:203]: I think it is automatic, from upgrading to a new version of webpack, they style.css file that rendered has different formatting.

File: package.json (hunk 3 above)
Comment (ramfox) [reply package.json:135]: kk!

**Pull request merged**